Mathlib/Analysis/InnerProductSpace/Positive.lean

```diff
@@ -211,65 +214,74 @@ end LinearMap
 
 namespace ContinuousLinearMap
 
-variable [CompleteSpace E] [CompleteSpace F]
-
-/-- A continuous linear endomorphism `T` of a Hilbert space is **positive** if it is self adjoint
+/-- A continuous linear endomorphism `T` of a Hilbert space is **positive** if it is symmetric
   and `∀ x, 0 ≤ re ⟪T x, x⟫`. -/
 def IsPositive (T : E →L[𝕜] E) : Prop :=
-  IsSelfAdjoint T ∧ ∀ x, 0 ≤ T.reApplyInnerSelf x
+  (∀ x y, ⟪T x, y⟫ = ⟪x, T y⟫) ∧ ∀ x, 0 ≤ T.reApplyInnerSelf x
 
-theorem IsPositive.isSelfAdjoint {T : E →L[𝕜] E} (hT : IsPositive T) : IsSelfAdjoint T :=
-  hT.1
+/-- In a complete space, a continuous linear endomorphism `T` is **positive** if it is
+symmetric and `∀ x, 0 ≤ re ⟪T x, x⟫`. -/
+theorem isPositive_def [CompleteSpace E] {T : E →L[𝕜] E} :
+    T.IsPositive ↔ IsSelfAdjoint T ∧ ∀ x, 0 ≤ T.reApplyInnerSelf x := by
+  simp [IsPositive, isSelfAdjoint_iff_isSymmetric, LinearMap.IsSymmetric]
 
-theorem IsPositive.inner_left_eq_inner_right {T : E →L[𝕜] E} (hT : IsPositive T) (x : E) :
-    ⟪T x, x⟫ = ⟪x, T x⟫ := by
-  rw [← adjoint_inner_left, hT.isSelfAdjoint.adjoint_eq]
+theorem IsPositive.isSymmetric {T : E →L[𝕜] E} (hT : T.IsPositive) :
+    T.IsSymmetric := hT.1
 
-theorem IsPositive.re_inner_nonneg_left {T : E →L[𝕜] E} (hT : IsPositive T) (x : E) :
-    0 ≤ re ⟪T x, x⟫ :=
-  hT.2 x
+theorem IsPositive.isSelfAdjoint [CompleteSpace E] {T : E →L[𝕜] E} (hT : IsPositive T) :
+    IsSelfAdjoint T := hT.isSymmetric.isSelfAdjoint
 
-theorem IsPositive.re_inner_nonneg_right {T : E →L[𝕜] E} (hT : IsPositive T) (x : E) :
-    0 ≤ re ⟪x, T x⟫ := by rw [inner_re_symm]; exact hT.re_inner_nonneg_left x
+theorem IsPositive.inner_left_eq_inner_right {T : E →L[𝕜] E} (hT : IsPositive T) (x y : E) :
+    ⟪T x, y⟫ = ⟪x, T y⟫ := hT.isSymmetric _ _
```

Comment:
Thanks for fixing this to have `x y` instead of just `x`! I noticed it was wrong in the diff above.